apps/docs/app/tokens/page.tsx

```diff
@@ -1,13 +1,18 @@
+"use client";
+
 import { allPages } from "@/.contentlayer/generated";
 import { Mdx } from "@/components/Mdx/MdxComponent";
+
 import "../page.css";
```

Comment:
> If there is a specific style for this page, bring it back to the same level, otherwise delete the import.

I followed the doc blindly, I will test it and remove it if not necessary.

Reply:
It seems indeed to work, removing it and if issues arises I'll investigate.